### PR TITLE
Nearsighted characters now CONFIRMED to always keep their glasses

### LINKED ISSUE #567: Poor Eyesight and ballistic goggles = no prescription glasses
## Description

As per the title

## Difference between expected and actual behavior

Like other cosmetics if it can't fit in the primary slot, it should spawn in the bag.

## Steps to reproduce

- choose poor eyesight
- choose ballistic goggles
- spawn without prescription goggles

## Issue bingo (not required to be all checked)

- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

makes sure that if you are nearsighted, no matter what other stuff you get in your loadout, you always keep a pair of glasses on you.

## Why It's Good For The Game

nearsighted characters should not be losing their glasses if they pick other loadout options, closes #567 

## Changelog

:cl:
fix: fixed nearsighted characters' glasses just deleting if they couldn't equip to your eyes. They will now try go in other spots in your inventory instead.
spellcheck: renamed "big sunglasses" to "shades" to be consistent with the character setup pref
add: added separate loadout options for prescription and non-prescription sunglasses, previously it was just prescription ones, also added a unique desc to prescription sunglasses.
add: added prescription healthhuds, sensorhuds, and sechuds, as well as sec sunglasses.
add: if you are nearsighted, and you spawn with a healthhud, sensorhud, or sechud, you will get prescription versions of said hud.
/:cl:

### CODE REVIEW

**Comment by MusselMain** [comment]
Can we make a rare variant for those who've got nearsighted and are playing a role like MP or Doctor? I generally don't use nearsighted, but I've seen enough players who're playing Doctor or medic, wearing glasses instead of healthHUD's/SensorMate's, that I have to mention this.

**Comment by stanalbatross** [comment]
> Can we make a rare variant for those who've got nearsighted and are playing a role like MP or Doctor? I generally don't use nearsighted, but I've seen enough players who're playing Doctor or medic, wearing glasses instead of healthHUD's/SensorMate's, that I have to mention this.

done in a different way - if you are nearsighted you will get prescription hud versions of your goggles

**Review by Shad0vvs — CHANGES_REQUESTED** [review]
About half of the sechuds are copy pasted without /perscription versions, please resolveo.

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -501,6 +502,10 @@
 	eye_protection = 1
 	hud_type = MOB_HUD_SECURITY_ADVANCED
 
+/obj/item/clothing/glasses/sunglasses/sechud/prescription
+	name = "Prescription Security HUD-Glasses"
+	desc = "Sunglasses wired up with the best nano-tech the USCM can muster out on the frontier. Displays information about any person you decree worthy of your gaze. Contains prescription lenses."
+prescription = TRUE
```
Comment (Shad0vvs) [inline code/modules/clothing/glasses/glasses.dm:508]:
```suggestion
    prescription = TRUE

```

File: code/modules/clothing/glasses/glasses.dm [hunk 2]
```diff
@@ -501,6 +502,11 @@
 	eye_protection = 1
 	hud_type = MOB_HUD_SECURITY_ADVANCED
 
+/obj/item/clothing/glasses/sunglasses/sechud/prescription
+	name = "Prescription Security HUD-Glasses"
+	desc = "Sunglasses wired up with the best nano-tech the USCM can muster out on the frontier. Displays information about any person you decree worthy of your gaze. Contains prescription lenses."
+    prescription = TRUE
```
Comment (Shad0vvs) [inline code/modules/clothing/glasses/glasses.dm:508]:
```suggestion
	prescription = TRUE
```

**Review by Shad0vvs — APPROVED** [review]

**Pull request merged**